highfidelity/hifi
# case 20058: Android - Handshake implementation

https://highfidelity.manuscript.com/f/cases/20058/User-on-mobile-unable-to-handshake

Handshake implementation:

**Test plan**

1. Login with two users, only one from Android is necessary for testing.
2. Preconditions required for Desktop handshake are also required on Android.
3. Press the right button with a handshake icon to start the process, avatars should be in front of each other. Avatars raise the hand as in Desktop.
4. After completing the process, both avatars should be connected.
